docs/guides/server/reverseproxy.adoc

```diff
@@ -233,6 +233,13 @@ to load additional certificates from headers `CERT_CHAIN_0` to `CERT_CHAIN_9` if
 | Whether the forwarded certificate is url-encoded or not. In NGINX, this corresponds to the `$ssl_client_cert` and `$ssl_client_escaped_cert` variables. This can also be used for the Traefik PassTlsClientCert middleware, as it sends the client certficate unencoded.
 |===
 
+[NOTE]
+====
+Consult the <@links.server id="configuration-provider"/> guide for details on the format of the options. For example you can use:
+
+<@kc.start parameters="--spi-x509cert-lookup-nginx-ssl-client-cert=SSL_CLIENT_CERT"/>
```

Comment:
I'm sorry, I realized the issue here is about using `-` instead of `--` as the delimiter in the SPI options. The `--` delimiter is the preferred one to prevent ambiguity. I originally misunderstood (and missed your comment) that you want to add a link to the providers config guide to help users understand better the format. I also then missed that in the example there was `-` instead of `--`. So it should be like this:
```suggestion
<@kc.start parameters="--spi-x509cert-lookup--nginx--ssl-client-cert=SSL_CLIENT_CERT"/>
```
You mentioned that didn't work for you?

Reply:
We can move this discussion to https://github.com/keycloak/keycloak/issues/43284#issuecomment-3381053430